### DISCUSSION TITLE
muduo 可以用于生产环境吗

### DISCUSSION THREAD

**Discussion by lkkey80** [chenshuo/muduo#604] [General]
GitHub issues are for tracking bugs, not for general discussing like a forum.

If you have a general question to ask, send it to mailing list:

muduo-library@googlegroups.com

Or visit:
https://groups.google.com/forum/#!forum/muduo-library

When file an issue muduo, please provide a [SSCCE](http://sscce.org):
Short, Self Contained, Correct (Compilable), Example.

Also specify the exact environment where the issue occurs:

**Comment by g302ge** [comment]
Of course , in this repo [Whale](https://github.com/Meituan-Dianping/octo-rpc/tree/master/whale) in Meituan-Dianping which uses Muduo

**Comment by Mr-whoz** [comment]
可以的，用muduo写了一套日志采集系统，目前部署在几百台机器上，为实时计算服务。

**Comment by fortunely** [comment]
我觉得更好的问题是：你在生产环境中，用muduo碰到了什么问题/bug？或者说，你的生产环境有什么特别需求，要看看muduo是否能满足。
